app/components/works/add_files_component.html.erb

```diff
@@ -13,7 +13,7 @@
                                   bs_toggle: 'modal',
                                   bs_target: '#contactUsModal' }) %>
       </p>
-        <div data-controller="dropzone" data-dropzone-max-file-size="10000" data-dropzone-max-files="1000">
+      <div data-controller="dropzone" data-dropzone-max-file-size="10000" data-dropzone-previews-container=".dropzone-files-previews">
```

Comment:
Here and below, dropzone controller is modified to support having multiple instances on the same page.